e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -719,6 +678,8 @@ $keyvaultKvps = @{
     "IOTHUB-USER-ASSIGNED-MSI-RESOURCE-ID" = $msiResourceId;
     "E2E-IKEY" = $instrumentationKey;
 
+    <# The connection strings below point to servers with incorrect TLS server certificates. Tests will attempt to connect and expect that the TLS connection ends in a security exception. #>
```

Comment:
Why wrapped in <> brackets?

Reply:
It doesn't need to be, it's a copy-paste mix-up.